src/Facebook/FacebookRequest.php

```diff
@@ -102,7 +102,8 @@ public function __construct(FacebookApp $app = null, $accessToken = null, $metho
         $this->setEndpoint($endpoint);
         $this->setParams($params);
         $this->setETag($eTag);
-        $this->graphVersion = $graphVersion ?: Facebook::DEFAULT_GRAPH_VERSION;
+        // @TODO Remove this fall-back
+        $this->graphVersion = $graphVersion ?: 'v2.7';
```

Comment:
Hello yguedidi! I have no certainties about this case version 2.8 yet and icoguinita for me, but I resolve to be with this team for sure has brilhates minds, we try faser tests, for and studying the case to come to perfection. I trust you guys the importate and not waste time, bad weather do quick pass worth every second. with an impeccable work. ok let's see what happens reversing from 2.7 to 2.8 ok